utils/cLog.cpp

```diff
@@ -1,262 +1,204 @@
-#include "../include/d2tmh.h"
 #include "cLog.h"
 
+#include "definitions.h"
+
+#include <cstdio>
+#include <ctime>
+#include <system_error>
+
+extern bool	bDoDebug;
+
+namespace {
```

Comment:
How /when do you decide to do this as opposed to have methods within the `private:` part of a class?

Reply:
If a function does not use any class member variables, it doesn't have to be tied to the object instantiated from the class. Two choices here: static member function, or a non-member function in the translation unit (.cpp file). That depends on if the function is called from outside of the translation unit. If it is: static member function. If not: in an anonymous namespace in the .cpp file. 
The reason is: information hiding. If we hide a function much as we can, it is restricted from where it can be called, which helps local reasoning and understanding the code.